CHANGELOG

```diff
@@ -1,3 +1,9 @@
+2.0.3 2015-03-27
+- Fix `graphite-web` install when using pip and default `root_dir`
+- Restart services when changing package versions
+- Improve acceptance and integration test coverage
+- Remove support for Ruby 1.8.7 and 1.9.2
```

Comment:
I don't think a minor bump is appropriate according to SemVer:

> MINOR version when you add functionality in a backwards-compatible manner

Happy to go with the patch bump since users can continue using previous versions with the old versions of Ruby.